src/theme.tsx

```diff
@@ -196,12 +196,64 @@ type LinkComponent = React.ComponentType<{
     children: React.ReactNode;
 }>;
 
-export const AnchorLink: LinkComponent = ({to, innerRef, ...props}) => (
+const AnchorLink: LinkComponent = ({to, innerRef, ...props}) => (
     <a ref={innerRef} href={typeof to === 'string' ? to : to?.pathname} {...props}>
         {props.children}
     </a>
 );
 
+const getReactRouter5Link =
+    (ReactRouterLink: React.ComponentType<any>): LinkComponent =>
+    ({...props}) =>
+        <ReactRouterLink {...props} />;
```

Comment:
isn't this equivalent?
```suggestion
    (ReactRouterLink: React.ComponentType<any>): LinkComponent => ReactRouterLink
```